lib/uffizzi/cli/preview.rb

```diff
@@ -76,33 +85,50 @@ def handle_list_command(project_slug)
     end
 
     def handle_create_command(file_path, project_slug)
-      hostname = ConfigFile.read_option(:hostname)
       params = file_path.nil? ? {} : prepare_params(file_path)
-      response = create_deployment(hostname, project_slug, params)
+      response = create_deployment(@hostname, project_slug, params)
 
       if ResponseHelper.created?(response)
-        handle_succeed_create_response(hostname, project_slug, response)
+        handle_succeed_create_response(project_slug, response)
+      else
+        ResponseHelper.handle_failed_response(response)
+      end
+    end
+
+    def handle_events_command(deployment, project_slug)
```

Comment:
I would rename `deployment` to `deployment_name`. It is not clear it is an object or not